README.md

```diff
@@ -180,7 +187,19 @@ If you want to see steps then you can use `--steps` option of codeception. Check
 
 **Note**:You can modify the timeout time by setting the value of **TIMEOUT** constant lower for fast machines and higher for slow computers. The constant located in the file `tests/codeception/acceptance/_bootstrap.php`
 
-Changing the browser the tests are running with?
+#### You can run an install before individual run `feature` using following command.
+
+**_Linux & Mac_**
+```bash
+$ tests/codeception/vendor/bin/robo run:install-and-test
+```
+
+**_Windows_**
+```cmd
+$ tests\codeception\vendor\bin\robo run:install-and-test
```

Comment:
I like it like it is. You do not need the whole command. If you type
`tests\codeception\vendor\bin\robo run:i`
the "installandtest" methode will be executed. 
But the long name explained exactly what the methode will do excetly.